packages/seacas/libraries/ioss/src/Ioss_Region.h

```diff
@@ -405,3 +409,80 @@ inline const std::vector<std::string> &Ioss::Region::get_qa_records() const
   IOSS_FUNC_ENTER(m_);
   return get_database()->get_qa_records();
 }
+
+namespace Ioss {
+
+template <typename T>
+bool verify_field_exists_in_entity_group(const std::string &field_name,
+                                         const std::vector<T*> &entity_container)
+{
+  bool found = false;
```

Comment:
This verifies that the field exists, but only verifies that the role is the same in a debug build.  Is it important whether the role is the same?  If so, then should be checked even in non-debug build.

Reply:
Yeah, it's important that they are the same so I'll make that check in all builds